dataset_access/conus404_explore.ipynb

```diff
@@ -269,35 +269,6 @@
     "da.hvplot(x='time', grid=True)"
    ]
   },
-  {
```

Comment:
I am suggseting we delete this, as it is covered in the temporal aggregation notebook. I noticed that you used `.mean()` here, and you use `.resample()` in the temporal aggregation notebook. Is one method better than the other?

Reply:
resample is better for time aggregation -- more flexible